helm/cluster-azure/templates/externaldns_private_app.yaml

```diff
@@ -38,9 +38,9 @@ metadata:
     giantswarm.io/managed-by: {{ .Chart.Name }}
 spec:
   catalog: default
-  name: external-dns
+  name: external-dns-app
   # repo: giantswarm/external-dns-app
-  version: 1.16.0
```

Comment:
Is there anything special that needs to be done when upgrading from 1.16.0 to 3.1.0?

Reply:
There was an issue in templating and we were not rendering this file properly. While fixing it, I noticed the version is also wrong. The version was 3.1.0 in the latest working version. See https://github.com/giantswarm/default-apps-azure/blob/main/helm/default-apps-azure/values.yaml#L211